app/views/employment/RemoveEmploymentView.scala.html

```diff
@@ -41,20 +44,29 @@
     @formWithCsrf(action = RemoveEmploymentController.submit(taxYear, employmentId)) {
         @heading(headingForUse, Some(messages("payeEmployment.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-3")
 
-        @if(lastEmployment) {
-            <p class="govuk-body">@messages("employment.removeEmployment.remove-all-paragraph")</p>
-        } else {
-
-            @if(isHmrcEmployment){
-                @govukInsetText(InsetText(
-                content = Text(messages("employment.removeEmployment.inset"))
-                ))
-            }
+        <div class="govuk-inset-text">
+            <h2  class="govuk-heading-m">
+                @{messages(employerName)} <br/>
+                @if(startDate.nonEmpty) {
+                    <span class="govuk-body-l">@{messages("employment.removeEmployment.started.date", dateFormatter(startDate).get)}</span>
+                }else {
+                    <span class="govuk-body-l">@{messages("common.started.date.missing")}</span>
+                }
+            </h2>
+        </div>
```

Comment:
The Period of employment styling is different to the prototype. I am not sure if it needs to change. If it doesn't I am happy to sign it off